### PR TITLE
Version 4.59

### PR DESCRIPTION
**Pull request by northwestwitch**
### Added
- [x] Button to go directly to HPO SV filter variantS page from case
- [x] `Scout-REViewer-Service` integration - show `REViewer` picture if available
- [x] Link to HPO panel coverage overview on Case page
- [x] Specify a confidence threshold (green|amber|red) when loading PanelApp panels
- [x] Functional annotations in variants lists exports (all variants)
- [x]  Cancer/Normal VAFs and COSMIC ids in in variants lists exports (cancer variants)
### Changed
- [x]  Better visualization of regional annotation for long lists of genes in large SVs in Variants tables
- [x] Order of cells in variants tables
- [x] More evident links to gene coverage from Variant page
- [x] Gene panels sorted by display name in the entire Case page
- [x] Round CADD and GnomAD values in variants export files
### Fixed
- [x] HPO filter button on SV variantS page
- [x] Spacing between region|function cells in SVs lists
- [x] Labels on gene panel Chanjo report
- [x] Fixed ambiguous duplicated response headers when requesting a BAM file from /static
- [x] Visited color link on gene coverage button (Variant page)

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by DN

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,16 +4,17 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
-## [unreleased]
+## [4.59]
 ### Added
 - Button to go directly to HPO SV filter variantS page from case
+- `Scout-REViewer-Service` integration - show `REViewer` picture if available
```
Comment (northwestwitch) [inline CHANGELOG.md:10]: 👍🏻

**Comment by northwestwitch** [comment]
I've deployed the latest on stage now

**Review by dnil — APPROVED** [review]
💯

**Comment by northwestwitch** [comment]
I've tested on stage all the latest merged PRs. I've found that exported excel files could be improved (see #3592). After this I think we are good with the new release!

**Comment by northwestwitch** [comment]
Deployed on cg-prod-services now: 🆗 
 
image

**Pull request merged**